metrics/cli/docs/features/update/06_update_feature.md

```diff
@@ -0,0 +1,97 @@
+Moreover, the CI Integrations tool (as a Metrics component) uses YAML configuration files to perform builds synchronization.
+
+To state that updating itself is possible, we should notice that both Firebase CLI and GCloud CLI have commands for updating the deployed projects with new values.
+And since the Metrics CLI already has the ability to interact with Firebase and GCloud CLIs for the deployment feature, the deployed components updating is possible as well.
+
+As it is possible to perform the required actions, we admit that it is possible to implement the Update feature.
+
+### Requirements
+> Define requirements and make sure that they are complete.
+
+The Feasibility study section defines the main actions the feature should perform.
+In this section, we should list the requirements and possible edge-cases that would cover the main actions of the Update feature.
+
+The `Update feature` has the following requirements:
+
+- for the parsing configuration action:
+  - should consume the configuration file path via the command line arguments;
+  - should read the configuration from the YAML configuration file.
+- for the redeploying action:
+  - should populate the Web Metrics Config variables;
+  - should stop the redeploy process of the Metrics CLI if something went wrong during the process;
+  - should redeploy/update the existing Metrics Web application to the same Firebase project;
+  - should skip the Sentry configuration steps if the Sentry attributes is missing in the YAML configuration.
+
+### Landscape
+> Look for existing solutions in the area.
+
+Before landscaping the feature, we should notice that the update feature is preliminary custom as implies changes to the Metrics CLI and is specific for Metrics project purposes.
+
+In general, to implement the update feature, we should implement the command, which should consistently perform the following actions:
+- parse the YAML configuration and extract data required for the update feature;
+- checkout new version of the Metrics Web application;
+- build the Flutter Web application;
+- configure Sentry for the application using the Sentry data obtained during parsing the YAML configuration;
+- redeploy the Metrics Web application to the existing Firebase project using the Firebase data obtained during parsing the YAML configuration.
+  
+To perform the first step, we're going to use the [yaml_map](https://github.com/Flank/flank-dashboard/tree/master/yaml_map) package.
+The package was chosen due to the following advantages over similar ones:
+- was used in the CI integrations tool, here is an [example](https://github.com/Flank/flank-dashboard/blob/0fadf00685df2b335def8c5697d6a5aa973ff92c/metrics/ci_integrations/lib/cli/config/parser/raw_integration_config_parser.dart#L19-L21);
+- parses the YAML file in the default dart [Map](https://api.dart.dev/stable/2.13.2/dart-core/Map-class.html).
+  
+And the last four steps, we're going to perform using the corresponding CLIs:
```

Comment:
🔨 I believe in this section, we should provide a landscape for the feature or its parts. More precisely, we should provide any examples, existing solutions, references, etc., that may be useful during designing this feature. But we should not choose the concrete approach we are to use to implement it. In this case, we can provide the usage examples of the third-party CLIs we are using for the deployment and underline that we are already using such an approach in the deploy command, which has similar functionality to the required.